script/csv_import/open_corporates/data/ground_truth.csv

```diff
@@ -537,7 +537,7 @@ us_mn;2545f2d3-93d4-e011-a886-001ec94ffe7f;52821;Elixery;united states;Minnesota
 us_ct;0904126;50934;People's United Financial;united states;Connecticut;;;850 Main Street Bridgeport, CT 06604 United States
 us_ny;1922485;50807;NASDAQ OMX Group;united states;New York;;;One Liberty Plaza 165 Broadway New York, NY 10006 United States
 us_ny;418186;923430;CA Technologies;united states;New York;;;520 Madison Avenue 22nd Floor New York, NY 10022 United States
-ge;A401954;8912;The Coca Cola Company;united states;GEORGIA;DELAWARE;us_de; ONE COCA COLA PLAZA ATLANTA GA 30313 404-676-2121
+us_ga;A401954;8912;The Coca Cola Company;united states;GEORGIA;DELAWARE;us_de; ONE COCA COLA PLAZA ATLANTA GA 30313 404-676-2121
```

Comment:
I assumed without verifying that Coca-Cola is based in Georgia (US) and not Georgia (Eurasia)